03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,22 @@
+1. Quickly explain how middleware is used in an Express application.
+
+Middleware is a function that have access to the request object, response object and next function in the application request-response cycle. The next function is a function in the Express router which, when invoked, executes the middleware succeeding the current middleware. 
+
+    Execute any code.
+    Make changes to the request and the response objects.
+    End the request-response cycle.
+    Call the next middleware in the stack.
+
+2. What does CRUD stand for?
+
+CRUD stands for Ceate, write, update and delete operations. 
```

Comment:
I'm sure this was a typo, but the R stands for read not write.

Reply:
Ooo yes. R stands for read.  Thanks a lot!